src/AddTodoForm.jsx

```diff
@@ -1,26 +1,41 @@
-import React, { useState } from 'react'
+import React, { useState } from "react";
 
-export default function AddTodoForm(props) {
-const [title, setTitle] = useState('');
+export default function AddTodoForm({addTodo}) {
```

Comment:
You don't need to change the prop name 'onAddTodo' to 'addTodo', you just have to change the value of 'onAddTodo' on App.jsx to 'addTodo' function defined at the top of the component in App.jsx [here](https://github.com/NataliaSokolova/React-todo/blob/5131da5e9b5979e2915e603e708ea1075eb3cf12/src/App.jsx#L9C1-L10C1)

Reply:
I will fix it. Thanks!